arxiv_completeness.py

```diff
@@ -1,10 +1,13 @@
 from datetime import date, datetime, timedelta
+import logging
 
 from errbot import BotPlugin, arg_botcmd
 from errcron import CrontabMixin
 
 from lib import arxiv_completness_check_script
 
+logging.basicConfig(level=logging.INFO)
```

Comment:
Please use module-level loggers `logger = logging.getLogger(__name__))` instead of the root logger to allow for more granular control over logging levels and handlers.